hedera-node/src/main/java/com/hedera/services/utils/SignedTxnAccessor.java

```diff
@@ -145,6 +145,7 @@ public SignedTxnAccessor(byte[] signedTxnWrapperBytes) throws InvalidProtocolBuf
 
 		txn = TransactionBody.parseFrom(txnBytes);
 		memo = txn.getMemo();
+		// should txnId accountID be parsed here ?
```

Comment:
We are lookingUp in the alias manager wherever we use accessor.getPayer() I think.